createdisk.sh

```diff
@@ -297,12 +297,12 @@ done
 crcBundleSuffix=crcbundle
 
 # libvirt image generation
-get_git_tag
+get_dest_dir
 
-if [ -z ${GIT_TAG} ]; then
+if [ -z ${DEST_DIR} ]; then
     destDirSuffix="$(date --iso-8601)"
 else
-    destDirSuffix="${GIT_TAG}"
+    destDirSuffix="${DEST_DIR}"
```

Comment:
get_dest_dir_suffix would be more accurate given the way it's used, but then I would also move this `if [ -z  ${DEST_DIR} ]` block in the helper function.